salt/personalised-covers/init.sls

```diff
@@ -64,6 +64,14 @@ personalised-covers-logs:
         - require:
             - file: personalised-covers-cache
 
+personalised-covers-data:
+    file.recurse:
+      - name: /srv/personalised-covers-data/formats
+      - source: salt://personalised-covers/data/formats
+      - user: {{ pillar.elife.webserver.username }}
+      - group: {{ pillar.elife.webserver.username }}
+      - dir_mode: 775
```

Comment:
could probably be stricter `555` and `444`? There should be no need to modify the files once they are deployed (maybe testing?)